diff --git a/SPECS/grub2.spec b/SPECS/grub2.spec
index 6d32e60..fa93878 100644
--- a/SPECS/grub2.spec
+++ b/SPECS/grub2.spec
@@ -228,10 +227,10 @@ GRUB_MODULES="	all_video boot btrfs cat chain configfile echo efifwsetup \
 		mdraid09 mdraid1x blscfg multiboot2 multiboot tftp"
 ./grub-mkimage -O %{grubefiarch} -o %{grubeficdname}.orig -p /EFI/BOOT \
 		-d grub-core ${GRUB_MODULES}
-%pesign -s -i %{grubeficdname}.orig -o %{grubeficdname} -a %{SOURCE1} -c %{SOURCE2} -n redhatsecureboot301
+%pesign -s -i %{grubeficdname}.orig -o %{grubeficdname} -a %{SOURCE1} -c %{SOURCE1} -n redhatsecureboot301
 ./grub-mkimage -O %{grubefiarch} -o %{grubefiname}.orig -p /EFI/%{efidir} \
 		-d grub-core ${GRUB_MODULES}
-%pesign -s -i %{grubefiname}.orig -o %{grubefiname} -a %{SOURCE1} -c %{SOURCE2} -n redhatsecureboot301
+%pesign -s -i %{grubefiname}.orig -o %{grubefiname} -a %{SOURCE1} -c %{SOURCE1} -n redhatsecureboot301
 cd ..
 %endif
 
